lib/catalog/martyrology.ts

```diff
@@ -2737,6 +2958,17 @@ export const Martyrology: { catalog: MartyrologyCatalog } = {
       titles: [Title.Virgin],
       dateOfDeath: 1912,
     },
+    // src:
+    // - mr_fr_1982_ed2_coutances
+    // - https://www.diocese50.fr/les-paroisses-et-les-doyennes/notre-dame-coutances/les-saints-du-diocese/sainte-marie-madeleine-postel-vierge
+    // - https://en.wikipedia.org/wiki/Marie-Madeleine_Postel
+    marie_madeleine_postel_virgin: {
```

Comment:
According to the [English](https://en.wikipedia.org/wiki/Marie-Madeleine_Postel) and [French](https://fr.wikipedia.org/wiki/Marie-Madeleine_Postel) Wikipedias, her religious name was `Marie-Madeleine`, first name `Julie Françoise-Catherine`, and surname `Postel`, therefore the key ID should be based on `Marie-Madeleine Julie Françoise-Catherine Postel`.

```suggestion
    marie_madeleine_julie_francoise_catherine_postel_virgin: {
```